src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 7, 0));
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//input[@id='product.dependencyMin']")
+	
+	//*[@id="product.dependencyMin"]
+	private WebElement enterMin;
+
+	/**
+	 * Method to Enter Price Rate.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterMin() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Min value");
+		JavaScriptExec.sleep();
+		WaitClass.WaitForElementisDisplay(driver, 10, enterMin);
+		Assert.assertTrue(enterPriceRate.isDisplayed());
+		//enterPriceRate1.clear();
+		enterMin.clear();
+		enterMin.sendKeys("1");
+
+}
+	@FindBy(how = How.XPATH, using = "//*[@id='dependency']/div[2]/div/table/tbody/tr/td[6]/a/img")
+	private WebElement clickDependencyPlusIcon;
+
+	/**
+	 * Method to click on dependency plus icon to add the product.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickDependencyPlusIcon() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("click on dependency plus icon to add the product.");
+		Assert.assertTrue(clickDependencyPlusIcon.isDisplayed());
+		clickDependencyPlusIcon.click();
+		JavaScriptExec.scrollToElementOnPage(driver, clickDependencyPlusIcon);
+	}
+	
+	@FindBy(how = How.XPATH, using = "//input[@name='product.descriptions[0].content']")
+	private WebElement enterEnglishDescription1;
+
+	/**
+	 * Method to Enter English Description.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterEnglishDescription1() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter English Description");
+		Assert.assertTrue(enterEnglishDescription.isDisplayed());
+		enterEnglishDescription.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 4));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='product.number']")
+	private WebElement enterProductCode1;
+
+	/**
+	 * Method to Enter Product Code.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterProductCode1() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Product Code");
+		Assert.assertTrue(enterProductCode.isDisplayed());
+		enterProductCode.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 4));
+
+	}
+	@FindBy(how = How.XPATH, using = "//input[@name='product.rate']")
+	private WebElement enterPriceRate1;
+
+	/**
+	 * Method to Enter Price Rate.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterPriceRate1() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Price Rate");
+		JavaScriptExec.sleep();
+		WaitClass.WaitForElementisDisplay(driver, 10, enterPriceRate);
+		Assert.assertTrue(enterPriceRate.isDisplayed());
+		//enterPriceRate1.clear();
+		enterPriceRate.sendKeys(BasePage.getCellData(xlsxName, sheetName, 6, 4));
```

Comment:
@mayankchhajer  remove this Wait and commented method

Reply:
Done